package/com.unity.formats.usd/Runtime/Scripts/Behaviors/UsdAsset.cs

```diff
@@ -443,6 +442,16 @@ public string usdFullPath
       if (!comp) { return; }
       Component.DestroyImmediate(comp);
     }
+    
+    /// <summary>
+    /// Clear internal data.
+    /// Call to <see cref="GetScene">GetScene()</see> to update them with the latest USD data.
+    /// </summary>
+    private void ClearLastData() {
+      m_lastScene = null;
```

Comment:
You and @lucillecaillaud should talk, I think her change to just call stage.Reload() may be a better option, though I'm not sure now if her change also requires clearing the prim map and access mask.